Koha/Plugin/Com/Biblibre/PatronImport/KohaPatron.pm

```diff
@@ -363,12 +363,20 @@ sub to_koha {
     }
 
     $this->{'borrowernumber'} = $borrowernumber;
+    my $patron_orm = Koha::Patron->find($borrowernumber);
```

Comment:
Find method should be used on Koha::Patrons (not Koha::Patron)

Reply:
😵 mais comment j'ai pu me tromper

Reply:
Branch repoussée